test/core/integration/concrete/lockup-linear/withdrawable-amount-of/withdrawableAmountOf.t.sol

```diff
@@ -3,32 +3,23 @@ pragma solidity >=0.8.22 <0.9.0;
 
 import { WithdrawableAmountOf_Integration_Concrete_Test } from
     "./../../lockup/withdrawable-amount-of/withdrawableAmountOf.t.sol";
-import { LockupLinear_Integration_Concrete_Test } from "./../LockupLinear.t.sol";
+import { LockupLinear_Integration_Shared_Test, Integration_Test } from "./../LockupLinear.t.sol";
 
 contract WithdrawableAmountOf_LockupLinear_Integration_Concrete_Test is
-    LockupLinear_Integration_Concrete_Test,
+    LockupLinear_Integration_Shared_Test,
     WithdrawableAmountOf_Integration_Concrete_Test
 {
-    function setUp()
-        public
-        virtual
-        override(LockupLinear_Integration_Concrete_Test, WithdrawableAmountOf_Integration_Concrete_Test)
-    {
-        LockupLinear_Integration_Concrete_Test.setUp();
-        WithdrawableAmountOf_Integration_Concrete_Test.setUp();
+    function setUp() public virtual override(LockupLinear_Integration_Shared_Test, Integration_Test) {
+        LockupLinear_Integration_Shared_Test.setUp();
     }
 
-    function test_GivenCliffTimeInFuture() external view givenSTREAMINGStatus {
+    function test_GivenCliffTimeInFuture() external givenSTREAMINGStatus {
```

Comment:
the "streaming" status is when the start time is greater than `block.timestamp`

https://github.com/sablier-labs/v2-core/blob/997444866a2f9279ffe4bdde59462e4598d6d87e/src/core/abstracts/SablierLockup.sol#L495-L503

> What is the status of a LL stream before the cliff time?

depends on the `block.timestamp`

---

since this a different discussion (not related to this PR) i'd suggest to  open a different discussion for this

Reply:
I see. The status is then `STREAMING` as long as `block.timestamp > startTime`. Then we should change the natspec over `modifier givenSTREAMINGStatus`. The line "In LockupLinear, the streaming starts after the cliff time" is confusing.

> since this a different discussion

How is this a different discussion?

Reply:
> The status is then STREAMING as long as block.timestamp > startTime. Then we should change the natspec over modifier givenSTREAMINGStatus. The line "In LockupLinear, the streaming starts after the cliff time" is confusing.

i see, i just copied the modifier (including the natspec) here, i didn't read the natspec tbh when moving it

https://github.com/sablier-labs/v2-core/blob/997444866a2f9279ffe4bdde59462e4598d6d87e/test/core/integration/shared/lockup/cancel.t.sol#L30-L36

fyi, it was written by you :))

![image](https://github.com/user-attachments/assets/3991ceb0-3ba4-4b52-ad1b-387417c4fa9f)


will remove it

> How is this a different discussion?

i tought you are saying to change the `statusOf` function, that's why i said it is a diff discussion